core/new-gui/src/app/dashboard/user/component/share-access/share-access.component.ts

```diff
@@ -66,7 +68,19 @@ export class ShareAccessComponent implements OnInit {
           this.validateForm.get("accessLevel")?.value
         )
         .pipe(untilDestroyed(this))
-        .subscribe(() => this.ngOnInit());
+        .subscribe(() => {
+          this.ngOnInit();
+          this.gmailService.sendEmail(
```

Comment:
It looks like you are not specifying the `email` field here. This would cause the email to be sent to the sender.

Reply:
If the receiver address is empty, the email will be sent to the currently logged-in user.

Reply:
Fixed.